include/boost/geometry/algorithms/detail/distance/segment_to_box.hpp

```diff
@@ -826,6 +879,12 @@ class segment_to_box<Segment, Box, 2, SBStrategy>
                          top_left, top_right, bottom_left, bottom_right,
                          sb_strategy);
         }
+        if (mirror)
+        {
+            strategy::distance::services::mirror<SBStrategy>::apply(result);
```

Comment:
The same story as with `swap`. What is mirrored here?

Reply:
ok, renamed to `mirror_points`